gatherling/models/Event.php

```diff
@@ -1027,6 +1027,26 @@ public static function getNextPreRegister($num = 20)
         return $events;
     }
 
+    public static function getStartingSoon($playername)
+    {
+        $db = Database::getConnection();
+        $stmt = $db->prepare('SELECT e.name FROM events e, entries n WHERE n.event_id = e.id AND n.player = ? AND prereg_allowed = 1 AND active = 0 AND finalized = 0 AND DATE_SUB(start, INTERVAL 0 MINUTE) < NOW() ORDER BY start');
```

Comment:
Why only prereg events?

Reply:
Ah yeah. Forgot that some events might not allow prereg. Revising it now

Reply:
On another note, the Your Upcoming Events will only show prereg events as well. Do we want to change it also? @silasary